packages/dds/tree/api-report/tree.api.md

```diff
@@ -1825,7 +1825,7 @@ export const TreeArrayNode: {
 
 // @public
 export interface TreeArrayNodeBase<out T, in TNew, in TMoveFrom> extends ReadonlyArray<T>, TreeNode {
-    insertAt(index: number, ...value: (TNew | IterableTreeArrayContent<TNew>)[]): void;
+    insertAt(index: number, ...value: readonly (TNew | IterableTreeArrayContent<TNew>)[]): void;
```

Comment:
this was a fix where the public API lacked read-only which was included in the implementation.

Reply:
Is there any reason not to add `readonly` to insertAtStart and insertAtEnd while you're at it?

Reply:
done.